requirements/mini.in

```diff
@@ -0,0 +1,2 @@
+matplotlib
```

Comment:
So plopp does not depend on Matplotlib, but cannot be used without it, e.g., for scatter plots?

Reply:
This must be because I still don't understand how `tox` works exactly.
We do have
```
install_requires =
    matplotlib
```
in the `setup.cfg`. Does this mean that I don't need to repeat it in the `tox` envs?

Reply:
Yes, you do not need to repeat package dependencies.